docs/DesignDocs/Attestation_API_Proposal.md

```diff
@@ -685,6 +685,109 @@ since legacy applications don't call these API functions explicitly.
  `oe_attester_shutdown()` and `oe_verifier_shutdown()` are NOP, so there
  is no need to call them upon application close.
 
+## Proposed V0.11 Implementation
+
+### Goals
+1. Consolidate SGX_ECDSA_P256 format IDs.
+2. Deprecate oe_report definitions.
+
+### Motivation
+The 2 extra SGX P256 evidence formats defined in V0.10 implementation can be reduced/deprecated to make the API surface cleaner:
+- `OE_FORMAT_UUID_SGX_ECDSA_P256_REPORT`
+- `OE_FORMAT_UUID_SGX_ECDSA_P256_QUOTE`
+
+For every listed format, `oe_get_evidence()` can wrap with
+an attestation header when the `OE_EVIDENCE_FLAGS_EMBED_FORMAT_ID` bit in its `flags` parameter is set. These are all possible SGX_ECDSA_P256 format structures that can be generated by oe_get_evidence():
+
+| Format ID | Evidence structure  |
```

Comment:
This table can't be displayed properly in github. For column divider, `\|\|` is needed instead of `||`.